packages/backend/src/util/handlers/createExpressSynthqlHandler.test.ts

```diff
@@ -0,0 +1,49 @@
+import { describe, expect, test } from 'vitest';
+import { createExpressSynthqlHandler } from './createExpressSynthqlHandler';
+import { queryEngine } from '../../tests/queryEngine';
+import { getMockReq, getMockRes } from 'vitest-mock-express';
+
+import { from } from '../../tests/generated.schema';
+import { DB } from '../../tests/db';
+
+describe('createExpressSynthqlHandler', () => {
```

Comment:
This test isn't really testing anything useful @jimezesinachi 

A few of the important invariants you should be testing:

1. The response is always 1..n lines of JSON
2. when returnLastOnly the status code can vary depending on the response kind
3. There's really no need to mock: spin up an actual server using `http.createServer`
4. Tests for error cases i.e. when the query is malformed
5. Add also tests using the react client. Notice that there is already a http.createServer in /react, instead of the custom handler code refactor to use the createExpressSynthqlHandler